src/main/java/org/spongepowered/api/potion/PotionEffect.java

```diff
@@ -25,7 +25,6 @@
 
 package org.spongepowered.api.potion;
 
-import org.spongepowered.api.entity.living.Living;
 import org.spongepowered.api.service.persistence.DataSerializable;
 
 /**
```

Comment:
We should add at least some javadocs on this class how to apply the effects.
For example `You can apply it to an entity using {@link Living#apply(PotionEffect)}.`